sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -528,7 +528,7 @@ public void testValidateReadSetsDefaultProject() throws Exception {
   }
 
   @Test
-  @Category(RunnableOnService.class)
+  @Category(NeedsRunner.class)
   public void testBuildSourceWithoutTableQueryOrValidation() {
     BigQueryOptions bqOptions = PipelineOptionsFactory.as(BigQueryOptions.class);
```

Comment:
You should also update this (and below) to use TestPipeline.testingPipelineOptions instead of PipelineOptionsFactory